pyproject.toml

```diff
@@ -34,7 +34,9 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 dev = [
-    "build"
+    "build",
+    "pylint",
+    "pylint-django",
```

Comment:
You need to update the pinned dependencies as well, by executing the `./tools/update_dependencies.sh` tool.